packages/react-components/src/collection/src/useCollection.ts

```diff
@@ -14,14 +14,14 @@ export interface CollectionItem {
     content: ElementType | ReactElement[];
     props: Record<string, any>,
```

Comment:
"," -> ";"

Reply:
I found a eslint rule that ensures this. I'll add it after the PRs